### ISSUE TITLE
Create Repo index of some kind

### ISSUE THREAD

**Issue by ff6347** [basiljs/examples#9]
Would be cool to have an Index or listing of all the examples in the README.md should be:

- autogenerated
- contain a description

**Comment by trych** [comment]
Good idea. Do you know any way to auto-generate it? Descriptions and so could be in inline comments and then parsed for auto-generation, I guess?

**Comment by ff6347** [comment]
If we Add Comments with a description to each we can parse it using:

- JSDoc http://usejsdoc.org/
- Documentation.js https://documentation.js.org/ (which we are using currently in basil.js)
I tried once do it by some rule, like "the first seven lines of each file" but this does not work at all.

Another Idea would be make it really browsable (using jekyll or something like this) but I need to think a bout how to do it and don't give it that high of a priority. A listing in the README with links and a short description should be enough for starters.
